quantuminspire/util/api/remote_runtime.py

```diff
@@ -112,13 +110,13 @@ async def _create_commit(api_client: ApiClient, algorithm: Algorithm) -> Commit:
         return await api_instance.create_commit_commits_post(obj)
```

Comment:
The description of the commit should be updated. Maybe just "Commit created by the SDK", since it is no longer a quantum cirquit.

Reply:
Done